x-pack/filebeat/cmd/main_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/elastic/beats/v7/testing/testflag"
+)
+
+func TestMain(m *testing.M) {
+	testflag.MustSetStrictPermsFalse()
+
+	flag.Parse()
```

Comment:
I'm not a fan of changing the global state like this, I rather fix the actual root cause.

Reply:
Other tests use the same approach, it's not new on the repo. Better than a unit test changing the file permission for good on the host